tokio-trace/src/level_filters.rs

```diff
@@ -0,0 +1,95 @@
+use std::cmp::Ordering;
+use tokio_trace_core::Level;
+
+/// Describes the level of verbosity of a span or event.
```

Comment:
Is this comment copied from `Level`? I think it should probably describe the difference between a `Level` and a `LevelFilter`...